tests/integration/goldens/asset/tests/unit/gapic/asset_v1/test_asset_service.py

```diff
@@ -206,7 +206,7 @@ def test__validate_universe_domain(client_class, transport_class, transport_name
     # TODO: This is needed to cater for older versions of google-auth
     # Make this test unconditional once the minimum supported version of
     # google-auth becomes 2.23.0 or higher.
-    google_auth_major, google_auth_minor, _ = [int(part) for part in google.auth.__version__.split(".")]
+    google_auth_major, google_auth_minor = [int(part) for part in google.auth.__version__.split(".")[0:2]]
```

Comment:
@parthea What's this change for?

Reply:
This is the fix for issue https://github.com/googleapis/gapic-generator-python/issues/1930. We don't use the minor version in the check so we don't need to parse it.